installing/disconnected_install/installing-mirroring-creating-registry.adoc

```diff
@@ -0,0 +1,43 @@
+:_content-type: ASSEMBLY
+[id="installing-mirroring-creating-registry"]
+= Creating a mirror registry
+include::modules/common-attributes.adoc[]
+:context: installing-mirroring-creating-registry
+
+toc::[]
+
+The _mirror registry for Red Hat OpenShift_ is a small and streamlined container registry that you can use as a target for mirroring the required container images of {product-title} for disconnected installations.
+
+If you already have a container image registry, such as Red Hat Quay, you can skip this section and go straight to xref:../../installing/disconnected_install/installing-mirroring-installation-images.adoc#installation-mirror-repository_installing-mirroring-installation-images[Mirroring the OpenShift Container Platform image repository].
+
+.Prerequisites
+
+* An {product-title} subscription.
+* {op-system-base-full} 8 with Podman 3.3 installed.
+* Fully qualified domain name for the Red Hat Quay service, which must resolve through a DNS server.
+* Passwordless `sudo` access on the target host.
+* Key-based SSH connectivity on the target host. SSH keys are automatically generated for local installs. For remote hosts, you must generate your own SSH keys.
+* 2 or more vCPUs.
+* 8 GB of RAM.
+* About 12 GB for {product-title} 4.10 Release images, or about 358 GB for {product-title} 4.10 Release images and {product-title} 4.10 Red Hat Operator images. Up to 1 TB per stream or more is suggested.
++
+[IMPORTANT]
+====
+These requirements are based on local testing results with only Release images and Operator images tested. Storage requirements can vary based on your organization's needs. Some users might require more space, for example, when they mirror multiple z-streams. You can use standard Red Hat Quay functionality to remove unnecessary images and free up space.
```

Comment:
@stevsmit this content was copied from your mirror registry files. I can see that "Release images" is capitalized a few times in the file, can you let me know if they do really need to be capitalized, or can they be lowered?

Reply:
I actually did this consciously because the dev team + Daniel made it look like it should always be capitalized (as in, any time they wrote Release images they capitalized it). That said, I just talked to our Principal dev and he thinks it should be lower case. I agree; I only ever did it because higher-stakes people _always_ capitalized it.